code/index.html

```diff
@@ -1,30 +1,257 @@
+      <img src="videos/ezgif.com-gif-maker-320.png" alt="dancing girl">
+    </picture>
 
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
+    <nav class="nav-container">
 
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+      <!-- logo -->
+      <svg class="logo" id="Capa_1" enable-background="new 0 0 512.019 512.019" height="512"
+        viewBox="0 0 512.019 512.019" width="512" xmlns="http://www.w3.org/2000/svg">
+        <g>
+          <path
+            d="m497.797 443.365-202.687-200.624v-14.889c43.104-16.1 72.301-57.464 72.301-104.349 0-61.426-49.974-111.4-111.4-111.4s-111.4 49.974-111.4 111.4c0 21.56 17.54 39.1 39.101 39.1 21.56 0 39.1-17.54 39.1-39.1 0-18.307 14.894-33.2 33.2-33.2s33.2 14.894 33.2 33.2c0 18.307-14.894 33.2-33.2 33.2-21.56 0-39.1 17.54-39.1 39.1v46.938l-202.742 200.674c-17.255 17.253-15.073 31.317-12.237 38.164s11.237 18.335 35.638 18.335l436.88.002c24.402 0 32.803-11.487 35.639-18.335 2.835-6.847 5.016-20.911-12.293-38.216zm-23.347 26.551-436.88-.002c-2.694 0-4.737-.235-6.206-.526.826-1.236 2.09-2.83 3.964-4.705l207.135-205.023c2.847-2.817 4.448-6.656 4.448-10.661v-53.196c0-5.018 4.082-9.1 9.1-9.1 34.849 0 63.2-28.351 63.2-63.2s-28.352-63.2-63.2-63.2-63.2 28.352-63.2 63.2c0 5.018-4.082 9.1-9.1 9.1-5.019 0-9.101-4.082-9.101-9.1 0-44.884 36.516-81.4 81.4-81.4s81.4 36.516 81.4 81.4c0 37.168-25.101 69.586-61.039 78.835-6.629 1.706-11.262 7.683-11.262 14.527v32.135c0 4.005 1.602 7.843 4.448 10.661l207.083 204.972c1.903 1.904 3.182 3.514 4.015 4.757-1.469.291-3.512.526-6.205.526z" />
+          <path
+            d="m266.617 286.321c-5.856-5.858-15.354-5.858-21.213 0l-135.565 135.563c-4.29 4.29-5.573 10.742-3.252 16.347 2.322 5.605 7.791 9.26 13.858 9.26h271.128c6.067 0 11.536-3.655 13.858-9.26 2.321-5.605 1.038-12.057-3.252-16.347zm-109.958 131.169 99.352-99.35 99.35 99.35z" />
+        </g>
+      </svg>
+
+      <!-- hamburger meny -->
+      <div class="hamburger" id="hamburger">
+        <div></div>
+        <div></div>
+        <div></div>
       </div>
 
-      <div class="question" id="section2">Section 2</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+      <!-- navbar -->
+      <ul>
+        <a class="nav-bar" href="">
+          <li>Lookbook</li>
+        </a>
+        <a class="nav-bar" href="">
+          <li>Trends</li>
+        </a>
+        <a class="nav-bar" href="">
+          <li>Pricing</li>
+        </a>
+        <a class="nav-bar" href="">
+          <li>Events</li>
+        </a>
+        <a class="nav-bar" href="">
+          <li>About us</li>
+        </a>
+      </ul>
+
+      <p class="sign-in">Sign in</p>
+    </nav>
+
+    <!-- center text and button on video -->
+    <article>
+      <h1>Feeling good?</h1>
+      <h3>Rent a dress!</h3>
+      <a class="button-container" href="#main">
+        <div class="btn">Get started</div>
+      </a>
+    </article>
```

Comment:
Perfect structure and nice that you're linking to the main content of the website 👏